recipes/tinyalsa/all/conanfile.py

```diff
@@ -1,68 +1,70 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.files import get, export_conandata_patches, apply_conandata_patches, chdir, copy, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.53.0"
 
 class TinyAlsaConan(ConanFile):
     name = "tinyalsa"
+    description = "A small library to interface with ALSA in the Linux kernel"
     license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/tinyalsa/tinyalsa"
     topics = ("tiny", "alsa", "sound", "audio", "tinyalsa")
-    description = "A small library to interface with ALSA in the Linux kernel"
-    exports_sources = ["patches/*",]
-    options = {"shared": [True, False], "with_utils": [True, False]}
```

Comment:
Why did you remove `with_utils` options? Is it not available for 1.1? Please, consider not removing an option, it affects directly the package ID: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-a-recipe

Reply:
Please, revert that change, people still can decide if they want or not executables installed too.

Reply:
I'll revert the change, though I'm still confused as to what utility it provides. The executables are tiny (combined total of 160 Kb on my machine). Removing them doesn't save any significant amount of time or space, and keeping them shouldn't break anything, the _only_ possible thing is the environment variables.